RecoTracker/IterativeTracking/python/MuonSeededStep_cff.py

```diff
@@ -113,82 +113,27 @@
-            name = 'muonSeededTracksInOutTight',
-            preFilterName = 'muonSeededTracksInOutLoose',
-            applyAdaptedPVCuts = cms.bool(False),
-            chi2n_par = 1.0,
-            minNumberLayers = 5,
-            min_nhits = 6,
-            maxNumberLostLayers = 3,
-            minNumber3DLayers = 2,
-            minHitsToBypassChecks = 10
-            ),
-        RecoTracker.FinalTrackSelectors.multiTrackSelector_cfi.highpurityMTS.clone(
-            name = 'muonSeededTracksInOutHighPurity',
-            preFilterName = 'muonSeededTracksInOutTight',
-            applyAdaptedPVCuts = cms.bool(False),
-            chi2n_par = 0.4,
-            minNumberLayers = 5,
-            min_nhits = 7,
-            maxNumberLostLayers = 2,
-            minNumber3DLayers = 2,
-            minHitsToBypassChecks = 20
-            ),
-        ) #end of vpset
-    ) #end of clone
-
-muonSeededTracksOutInSelector = RecoTracker.FinalTrackSelectors.multiTrackSelector_cfi.multiTrackSelector.clone(
-    src='muonSeededTracksOutIn',
-    trackSelectors= cms.VPSet(
-        RecoTracker.FinalTrackSelectors.multiTrackSelector_cfi.looseMTS.clone(
-            name = 'muonSeededTracksOutInLoose',
-            applyAdaptedPVCuts = cms.bool(False),
-            chi2n_par = 10.0,
-            minNumberLayers = 3,
-            min_nhits = 5,
-            maxNumberLostLayers = 4,
-            minNumber3DLayers = 0,
-            minHitsToBypassChecks = 7
-            ),
-        RecoTracker.FinalTrackSelectors.multiTrackSelector_cfi.tightMTS.clone(
-            name = 'muonSeededTracksOutInTight',
-            preFilterName = 'muonSeededTracksOutInLoose',
-            applyAdaptedPVCuts = cms.bool(False),
-            chi2n_par = 1.0,
-            minNumberLayers = 5,
-            min_nhits = 6,
-            maxNumberLostLayers = 3,
-            minNumber3DLayers = 2,
-            minHitsToBypassChecks = 10
-            ),
-        RecoTracker.FinalTrackSelectors.multiTrackSelector_cfi.highpurityMTS.clone(
-            name = 'muonSeededTracksOutInHighPurity',
-            preFilterName = 'muonSeededTracksOutInTight',
-            applyAdaptedPVCuts = cms.bool(False),
-            chi2n_par = 0.4,
-            minNumberLayers = 5,
-            min_nhits = 7,
-            maxNumberLostLayers = 2,
-            minNumber3DLayers = 2,
-            minHitsToBypassChecks = 20
-            ),
-        ) #end of vpset
-    ) #end of clone
+
+# Final Classifier
+from RecoTracker.FinalTrackSelectors.TrackCutClassifier_cfi import *
+muonSeededTracksInOutClassifier = TrackCutClassifier.clone()
+muonSeededTracksInOutClassifier.src='muonSeededTracksInOut'
+muonSeededTracksInOutClassifier.mva.minPixelHits = [0,0,0]
+muonSeededTracksInOutClassifier.mva.maxChi2 = [9999.,9999.,9999.]
+muonSeededTracksInOutClassifier.mva.maxChi2n = [10.0,1.0,0.4]
+muonSeededTracksInOutClassifier.mva.minLayers = [3,5,5]
+muonSeededTracksInOutClassifier.mva.min3DLayers = [1,2,2]
```

Comment:
inOut and outIn cuts need to be re-tuned.
in particular inOut starts from tracks already found. for those min3DLayers better be >1 even for loose.